### ISSUE TITLE
Setting up on windows 11 does not work? 

### ISSUE THREAD

**Issue by ong6** [MarkBind/markbind#1721]
**Tell us about your environment**
Windows 11

* **MarkBind Version:**
Current Release

**What did you do? Please include the actual source code causing the issue.**

I noticed that the startup guide does not request users to run `npm i` after the `npm link` command. 
![image](https://user-images.githubusercontent.com/19247856/149660209-68b4874a-68f9-4133-9dc6-ad8d215ba94a.png)

When following the steps to set up my environment on another computer I realized that if I don't run `npm i` and run `npm run setup` I get the following error:
![image](https://user-images.githubusercontent.com/19247856/149660259-83cfd6f3-e869-4965-af71-dd95fa9d5afd.png)

Basically, it just does not work properly. But after running `npm i` in the root dir, the `npm run setup` works.

I'm not sure why this occurred as well but I guess we add that users may have to run `npm i` before `npm run setup` in the docs? 

The log: [2022-01-16T12_31_53_544Z-debug.log](https://github.com/MarkBind/markbind/files/7876949/2022-01-16T12_31_53_544Z-debug.log)

**Comment by ong6** [comment]
My suggestion: Just add a additional step of running `npm i` in the docs as that seemed to fix my issues. But investigating this issue would be a good idea also.

**Comment by jonahtanjz** [comment]
Thanks for raising this issue @ong6

I've tested this in Windows 10 and it seems `npm run setup` can complete without any issues. This may be a Windows 11 specific issue.

> Just add a additional step of running npm i in the docs as that seemed to fix my issues. But investigating this issue would be a good idea also.

`npm run setup` will run [`npm ci`](https://docs.npmjs.com/cli/v8/commands/npm-ci) instead of `npm intall`. The difference is that `npm ci` will install the dependencies from `package-lock.json` and validate using `package.json` (will throw an error if dependencies do not match) while `npm i` will install dependencies using `package.json` and possibly write to `package-lock.json` if there are any differences in the versions installed.

My guess is that there is a dependency's version that is not compatible with Windows 11 and this causes `npm ci` to throw an error. `npm i` might resolve the issue as it tries to find a compatible version but will need to be careful as this may overwrite the dependency's version stated in `package-lock.json` (which will affect the version installed when someone else runs `npm run setup`).

**Comment by ong6** [comment]
Thanks for the quick reply @jonahtanjz! I shall make a PR with a documentation update for the people who accidentally updated to windows 11 😄.
